x-pack/plugins/observability/common/co_pilot.ts

```diff
@@ -221,6 +228,73 @@ export const coPilotPrompts = {
       ];
     },
   }),
+  [CoPilotPromptId.InfraExplainProcess]: prompt({
+    params: t.type({
+      command: t.string,
+    }),
+    messages: ({ command }) => {
+      return [
+        INFRA_SYSTEM_MESSAGE,
+        {
+          content: `I am a software engineer. I am trying to understand what a process running on my
```

Comment:
@SeanHeelan I removed the word "Linux" from here as the process metricset [supports other OS](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-system-process.html)'.  I tested it using a windows command and it was able to provide context in the response that it is windows.  Ideally, I think we'd pass in the OS, but might do that in a followup.